### PR TITLE
chore: enable Crowdin for WireReusableUIComponents - WPB-9727

### PR DESCRIPTION
**Pull request by caldrian**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-9727" title="WPB-9727" target="_blank">TaskWPB-9727</a>  Implement UISplitViewController for iPad
  </td></table>
  <br />
 

### Issue

Currently strings are only localised for the app target.
This PR reconfigures everything regarding Crowdin, so that we can have strings in the WireReusableUIComponents target.

---

### Checklist

- [x] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [x] Description is filled and free of optional paragraphs.
- [x] Adds/updates automated tests.

---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
- [x] Make sure you use the API for UI elements that support large fonts.
- [x] All colors are taken from WireDesign.ColorTheme or constructed using WireDesign.BaseColorPalette.
- [x] New UI elements have Accessibility strings for VoiceOver.

### CODE REVIEW

File: crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Credentials.
+
```
Comment (caldrian) [inline crowdin.yml:2]: GitHub shows it as new file, however, the file is actually moved and the paths are adjusted.

File: scripts/generate-licenses.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 set -Eeuo pipefail
 
 # Wire
-# Copyright (C) 2023 Wire Swiss GmbH
+# Copyright (C) 2024 Wire Swiss GmbH
```
Comment (caldrian) [inline scripts/generate-licenses.sh:5]: I can move these changes into a different PR if you prefer.

File: WireUI/Sources/WireReusableUIComponents/Resources/Base.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+// Sidebar
+"sidebar.conversation_filter.title" = "Conversations";
```
Comment (caldrian) [inline WireUI/Sources/WireReusableUIComponents/Resources/Base.lproj/Localizable.strings]: A placeholder string.

File: WireDomain/Package.swift [hunk 4]
```diff
@@ -5,7 +5,6 @@ import PackageDescription
 
 let package = Package(
     name: "WireDomainPackage",
-    defaultLocalization: "en",
```
Comment (caldrian) [inline WireDomain/Package.swift:8]: fix, according to the [docs](https://developer.apple.com/documentation/xcode/localizing-package-resources):
"When you declare a value for defaultLocalization in the package manifest, Xcode requires the package to contain localized resources."

**Review by johnxnguyen — APPROVED** [review]
looks good, thanks @caldrian

File: crowdin.yml (hunk 1 above)
Comment (samwyndham) [reply crowdin.yml:2]: I hate it when this happens. I've tried fixing this by doing moves in their own commit and changes in another etc but GitHub often misses my intention 😠

**Review by samwyndham — COMMENTED** [review]
Looks good to me 🌐

**Review by samwyndham — APPROVED** [review]

**Review by jullianm — APPROVED** [review]

**Pull request merged**